Management/Repository/CommentRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Management.DomainModels;
+using Management.Interface;
+using Optional;
+using Storage.Interface;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Data.Common;
+using DomainComment = Management.DomainModels.Comment;
+using StorageComment = Management.StorageModels.Comment;
+
+namespace Management.Repository
+{
+    public class CommentRepository : ICommentRepository
+    {
+        private const string _tableName = "comment";
+
+        private const string _keyColumnName = "uniqueId";
+        
+        private readonly IRepository _repository;
+
+        public CommentRepository(IRepository repository)
+        {
+            _repository = repository ?? throw new ArgumentNullException(nameof(repository));
+            // Create the table if not exist.
+            // - To start mysql server:
+            // cd /c/Program\ Files/MySQL/MySQL\ Server\ 8.0/bin/
+            // mkdir ../data (if it's the first time)
+            // ./mysqld.exe
+            // - To check if mysql server is started:
+            // net start | grep MySQL
+            // - To connect to mysql:
+            // cd /c/Program\ Files/MySQL/MySQL\ Server\ 8.0/bin/
+            // mysql.ext -uroot -pasesharp
+            // password is asesharp
+            Console.WriteLine("@mli: Before calling CreateTable...");
+            _repository.CreateTable(_tableName,
+                new Tuple<string, string>("uniqueId", "VARCHAR(100)"),
+                new List<Tuple<string, string>>() {
+                    new Tuple<string, string>("userId", "VARCHAR(100)"),
+                    new Tuple<string, string>("country", "VARCHAR(50)"),
+                    new Tuple<string, string>("state", "VARCHAR(50)"),
+                    new Tuple<string, string>("createdAt", "VARCHAR(50)"),
+                    new Tuple<string, string>("commentStr", "VARCHAR(500)")}).Wait();
```

Comment:
This is probably not a good place to put `CreateTable`, since this object seems to be constructed per request instead of per service :(

Not sure if we can use the approach in https://stackoverflow.com/questions/38532764/create-tables-on-runtime-in-ef-core?

Reply:
This will actually not create a new table everytime since there's `IF NOT EXISTS` in the query. It will just have an additional cost of checking whether the table exists for every request, which is not very good.
Will assume the table is already created before service start then.